docs/cow-protocol/tutorials/cow-swap/flash-loans.mdx

```diff
@@ -0,0 +1,266 @@
+---
+sidebar_position: 5
+---
+
+# Flash loans
+
+We’ll walk through a concrete example of a debt repayment using flash loans on Sepolia.
+
+There is an [example order](https://explorer.cow.fi/sepolia/orders/0x86ff28a37bf70c549edfe753d899d44307b1b4c3ae943c6f5bdfe271942fd4f135ed9a9d1122a1544e031cc92fcc7ea599e28d9c67dc66ab) for this particular case on Sepolia, including the [corresponding on-chain transaction](https://sepolia.etherscan.io/tx/0x6f4fc9b450e08dab0ab2d935e7c7dcade9e44e330e484657e5b1fadb6c6e8851).
+The example was executed using an [automated script](https://github.com/mrnaveira/cow-flashloan-tester) that places and signs a flash loan order on the Sepolia network. The following sections of the tutorial include code snippets from this tool.
+
+## Setup
+The setup for the flash loan example is as follows:
+* A Safe wallet at address 0x35eD9A9D1122A1544e031Cc92fCC7eA599e28D9C. We use a smart contract wallet so that smart contracts can initiate the withdrawal of the collateral token.
+* The Safe borrows 10,000 USDT on Aave using 500,000 USDC as collateral.
+* Aave is used as the flash loan lender.
+* The necessary `ERC-20` approvals are set in the wallet — to CoW's Vault Relayer contract and to the relevant Aave pool contracts.
+* Some non-zero amount of the order's buy token in the Safe's balance, to pass CoW Swap order validation.
```

Comment:
I guess most of the readers would look at this sentence and say "what, why?", but I don't have a suggestion how to improve it, maybe adding a bit more context here (but that would bloat the core idea around flashloans we are trying to explain here)

Reply:
Makes sense. I added just a tiny bit of context (_spam protection measure_).